contracts/state.cairo

```diff
@@ -360,6 +365,20 @@ func get_pool_volatility_auto{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, ra
 }
 
 
+@view
+func get_max_lpool_balance{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    pooled_token_addr: Address) -> (max_balance: Uint256) {
+        let(maxbal) = max_lpool_balance.read(pooled_token_addr);
+        return (maxbal, );
+}
+
+@external
```

Comment:
should not be external without `Proxy.assert_only_admin();`

also, create one `set_max_lpool_balance` that is not external and is used in the add_lptoken... and another one `set_max_lpool_balance_external` that is used for updating existing pools...

similarly to how `set_pool_volatility_adjustment_speed_external` is done

Reply:
also i don't see the function in the interface

Reply:
interface may be behind

Reply:
please take a look into this